### PR TITLE
Check if builtin server is active

### PR DESCRIPTION
**Pull request by barryvdh**
This adds the check as discussed in https://github.com/mnapoli/bref-bootstrap-benchmarks/issues/10#issuecomment-445453011

Seems to be working. Shouldn't affect hot performance though.

### CODE REVIEW

File: f/php/bootstrap [hunk 1]
```diff
@@ -25,15 +27,29 @@ $server->start(function ($type, $output) {
...
 register_shutdown_function(function() use($server) {
     $server->stop();
 });
 
+$wait = 10000; // 10ms
+$timeout = 5000000; // 5 secs
+$elapsed = 0;
+
+do {
+    usleep($wait);
```
Comment (staabm) [inline f/php/bootstrap]: why do we need this sleep here? the fsockopen() is a blocking operation and will return as soon as the socket is available, no?

Comment (barryvdh) [reply f/php/bootstrap]: Hmm, it didn't in my test (on mac). It just returned directly. Does it work for you?

Comment (staabm) [reply f/php/bootstrap]: hm maybe the socket will not block until the server side socket is available.

I did not test it

Comment (staabm) [reply f/php/bootstrap]: did you check whether the script can find the server now faster then with a 1sec wait?

Comment (barryvdh) [reply f/php/bootstrap]: I was also expecting guzzle to wait, but it disnt. Perhaps because it was refused directly.

**Comment by mnapoli** [comment]
Thanks!

**Pull request merged**